docs/_data/cards.yml

```diff
@@ -121,32 +122,34 @@
   passives:
     - name: Pure Angel
       levels:
-        - When using the blue orb, restores <b>1</b> additional MP.
-          All allies gain a barrier absorbing <b>10%</b> of maximum HP for 6 turns.
-        - When using the blue orb, restores <b>2</b> additional MP.
-          All allies gain a barrier absorbing <b>15%</b> of maximum HP for 6 turns.
-        - When using the blue orb, restores <b>3</b> additional MP.
-          All allies gain a barrier absorbing <b>20%</b> of maximum HP for 6 turns.
-    - name: 天使の爛漫
+        - When using the blue orb, generates <b>1</b> additional MP.
+          All allies gain a barrier that absorbs up to <b>10%</b> of maximum HP for 6 turns.
+        - When using the blue orb, generates <b>2</b> additional MP.
+          All allies gain a barrier that absorbs up to <b>15%</b> of maximum HP for 6 turns.
+        - When using the blue orb, generates <b>3</b> additional MP.
+          All allies gain a barrier that absorbs up to <b>20%</b> of maximum HP for 6 turns.
+    - name: Angel in Bloom
       levels:
         - When any ally receives a critical hit, they restore <b>10%</b> of the damage taken, then gain <b>1%</b> critical damage reduction (stacking up to 10 times).
         - When any ally receives a critical hit, they restore <b>12.5%</b> of the damage taken, then gain <b>2%</b> critical damage reduction (stacking up to 10 times).
         - When any ally receives a critical hit, they restore <b>15%</b> of the damage taken, then gain <b>3%</b> critical damage reduction (stacking up to 10 times).
-
+  notes:
+    Strongest Ally: Witch with the highest offensive stat total (Physical + Magic).
+    Weakest Ally: Witch with the lowest current HP stat.
```

Comment:
I've used strongest/weakest because they flow better, but you're right that annotating exactly what that means would be good. Not sure if it's better in notes or just giving up and using those descriptions inline.

Consistency corrections:

- Strongest **Ally** -> Strongest ally
- Same for weakest
- **physical** (lowercase) and **magical** (lowercase, -al)

Reply:
Also, there's more than just here that this terminology gets used, so probably worth doing a pass over the whole set.

Reply:
i think doing it inline with the full words is the better option since strongest/weakest implies the same/similar comparison when its not (offensive stats vs hp), especially since in this case its put in the same paragraph

and also because lavie uses the same thing and will probably continue using the same thing (and i am definitely not biased at all)

side note, anything else needs changing before you can push this update? i think we want to push this before the next banner comes, then ill drop iv klore and konohana tls along with it

Reply:
> i think doing it inline with the full words is the better option since strongest/weakest implies the same/similar comparison when its not (offensive stats vs hp), especially since in this case its put in the same paragraph
Agreed. Wording as "ally/allies with the highest/lowest (current HP/combined physical and magic attack)" seems concise enough.

> anything else needs changing before you can push this update?
I was going to say the consistency corrections above, but I think I'll do a pass over the current cards with the thoughts above, so I'd change it anyway.